src/index.js

```diff
@@ -2,16 +2,33 @@ import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App.js";
 import { Provider } from "react-redux";
-import { createStore } from "redux";
-import rootReducer from "./redux";
+import { createStore} from "redux";
+import rootReducer, { setMode, setSelectedCSV } from "./redux";
+import { allDatasets } from "./datasetManifest";
+import { parseCSV } from "./csvReaderWrapper";
 
 export const store = createStore(rootReducer);
 
 export const initAll = function(options) {
+  const mode = options && options.mode;
+
+  setMode(mode);
```

Comment:
Do you need to do `store.dispatch(setMode(mode));` or is this enough?

Reply:
Will fix!